system/i18n/cms_de.properties

```diff
@@ -70,6 +70,7 @@ datamanager.addrecord.title={1} hinzuf\u00FCgen
 datamanager.addrecord.breadcrumb.title=Datensatz hinzuf\u00FCgen
```

Comment:
Please do not provide German translations this way. We need to do this through Zanata and our translation processes (you do not need to worry about it)